src/servers/graphql-old-server.ts

```diff
@@ -228,12 +229,24 @@ const resolvers = {
         return true
       },
     }),
-    noauthAddInvoice: async (_, { username, value }, { logger }) => {
```

Comment:
Double-checking that the `{ logger }` arg is ok to remove here. I didn't quite understand how this got here from reading the schema, but it's no longer needed in the function body (cc @samerbuna)

Reply:
you can just remove it: `(_, { username, value })`.